### PR TITLE
Growing Dread -- might need support

### PR DESCRIPTION
**Pull request by Fulgur14**

### CODE REVIEW

**Comment by tool4ever** [comment]
I could not find any cards that would make a different player than the activator turn it face up (on the battlefield)
So this might be enough for now

**Comment by Fulgur14** [comment]
I know, but with return to Tarkir lined up, it's something that might matter later.

File: forge-gui/res/cardsfolder/h/hallow.txt [hunk 1]
```diff
@@ -1,9 +1,11 @@
...
+A:SP$ Effect | ValidTgts$ Card | TargetType$ Spell | TgtZone$ Stack | TgtPrompt$ Select target spell to prevent damage from | ReplacementEffects$ PreventDmg | Triggers$ TargetMoved | RememberObjects$ TargetedSource | SpellDescription$ Prevent all damage target spell would deal this turn. You gain life equal to the damage prevented this way.
 SVar:PreventDmg:Event$ DamageDone | ValidSource$ Card.IsRemembered | ReplaceWith$ GainLifeYou | PreventionEffect$ True | Description$ Prevent all damage that would be dealt by targeted spell this turn. You gain life equal to the damage prevented this way.
 SVar:GainLifeYou:DB$ GainLife | Defined$ You | LifeAmount$ X
 SVar:X:ReplaceCount$DamageAmount
+SVar:TargetMoved:Mode$ ChangesZone | ValidCard$ Card.IsRemembered | ExcludedDestinations$ Battlefield | Execute$ ExileEffect | TriggerZones$ Command | Static$ True
```
Comment (Hanmac) [inline forge-gui/res/cardsfolder/h/hallow.txt:8]: `TriggerZones$ Command` is not needed, because this is already set for EffectEffect

**Review by Hanmac — APPROVED** [review]
i wish Hallow could be coded better, but that is good enough for now

**Pull request merged**